tests/Jenkins/utilities.sh

```diff
@@ -252,9 +252,16 @@ installDIRAC() {
       pip install DIRAC "${DIRAC_RELEASE}"
     fi
   fi
-  for module_path in "${ALTERNATIVE_MODULES[@]}"; do
-    pip install ${PIP_INSTALL_EXTRA_ARGS:-} "${module_path}"
-  done
+
+  if [[ -n "${INSTALLATION_BRANCH}" ]]; then
+    # Use this for (e.g.) running backward-compatibility tests
+    echo "pip-installing DIRAC from git+https://github.com/DIRACGrid/DIRAC.git@${INSTALLATION_BRANCH}#egg=DIRAC[client]"
+    pip install "git+https://github.com/DIRACGrid/DIRAC.git@${INSTALLATION_BRANCH}#egg=DIRAC[client]"
```

Comment:
How about moving the URL to the CI config so it could also work with forks or CERN GitLab:
```suggestion
    pip install "${PIP_INSTALLATION_TARGET}"
```

Reply:
Then what is in install_client.sh in https://github.com/DIRACGrid/DIRAC/pull/6654/files would need to do a string manipulation. That's not per-se a problem, but anyway to get test code from different locations (e.g. for DIRAC and for an extension) we would need to pass more than 1 URL. I don't want to embark in coding such a thing right now, but I can think about it later on.